packages/server/package.json

```diff
@@ -20,6 +20,8 @@
     "start:storybook": "start-storybook -p 6006 --no-dll"
   },
   "dependencies": {
+    "@fontsource/material-icons": "^4.4.5",
+    "@fontsource/roboto": "^4.4.5",
```

Comment:
I think these can be devDependencies? the frontend source is *built* and published with the npm package

Reply:
Yep! Moved them over in both viewer/server.